pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -195,7 +195,7 @@ func TestCreateNopoolFilesystem(t *testing.T) {
 	validateStart(t, context, fs)
 
 	// Test multiple filesystem creation
-	executor = &exectest.MockExecutor{
+	_ = &exectest.MockExecutor{
```

Comment:
You can remove this whole code block from lines 197-205, looks like it's not used

Reply:
Done.